nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/scalar/LeakyReLU.java

```diff
@@ -50,12 +50,22 @@ public LeakyReLU(SameDiff sameDiff, SDVariable i_v, boolean inPlace, double alph
 
     }
 
+    public LeakyReLU(SameDiff sameDiff, SDVariable i_v, double alpha) {
+        this(sameDiff, i_v, false, alpha);
+    }
+
     public LeakyReLU(SameDiff sameDiff, SDVariable i_v, Object[] extraArgs, double alpha) {
         super(sameDiff, i_v, alpha, extraArgs);
         this.alpha = alpha;
         this.extraArgs = new Object[]{alpha};
     }
 
+    public LeakyReLU(SameDiff sameDiff, SDVariable i_v, SDVariable alpha) {
+        super(sameDiff, i_v, alpha.eval().getDouble(0), false);
+        double dAlpha = alpha.eval().getDouble(0);
+        this.alpha = dAlpha;
+    }
```

Comment:
alpha should be double (arg) not SDVariable (input). Update kotlin definition.
FYI this sort of thing isn't safe in general (only works for constants which an arbitrary SDVariable may not be - alpha.eval().getDouble(0))

Reply:
Fixed.